satpy/readers/fci_l1c_nc.py

```diff
@@ -31,6 +31,14 @@
     (FDHSI) files. Support for High Spatial Resolution Fast Imagery (HRFI) files
     will be implemented when corresponding test datasets will be available.
 
+
+.. note::
+    For reading compressed data, the `hdf5plugin` package needs to be installed:
+        pip install hdf5plugin
+    or:
+        conda install hdf5plugin -c conda-forge
```

Comment:
Does this still show up as bold for you? If so, try adding a blank line between the `note` line and the actual message.

Reply:
Yes it does. Blank line didn't change it to normal type :thinking: